src/__tests__/octokit.test.ts

```diff
@@ -385,5 +421,6 @@ describe("getTeamsForRepositoriesInOrganisation", () => {
     expect(actualTeamsForRepositoriesInOrganisation).toEqual(
       expectedTeamsForRepositoriesInOrganisation
     );
+    vi.restoreAllMocks();
```

Comment:
See above, but this one's particularly insidious because it won't happen if the expectation above fails.